semantic-source/semantic-source.cabal

```diff
@@ -43,15 +46,13 @@ library
     Source.Range
     Source.Source
     Source.Span
-  -- other-modules:
-  -- other-extensions:
   build-depends:
       aeson          ^>= 1.4.2.0
     , base            >= 4.12 && < 5
     , bytestring     ^>= 0.10.8.2
     , deepseq        ^>= 1.4.4.0
     , generic-monoid ^>= 0.1.0.0
-    , hashable       ^>= 1.2.7.0
+    , hashable        >= 1.2.7 && < 1.4
```

Comment:
`hashable` 1.3 supports `ghc` 8.8.1, 1.2.x does not. The [breaking changes](https://hackage.haskell.org/package/hashable-1.3.0.0/changelog) in 1.3 don’t impact us.